velox/exec/RowContainer.h

```diff
@@ -506,6 +562,35 @@ class RowContainer {
     }
   }
 
+  template <typename T>
+  static void extractValuesOffsetsWithNulls(
+      const char* const* rows,
+      const BufferPtr& offsets,
```

Comment:
let's rename offsets of rowNumbers and columnOffset to offset for consistency

Reply:
I changed the usage of offsets to rowNumbers. 

Most of these functions also have a resultOffset, so I had used columnOffset to make the usage clear. wdyt ?